docs/configurations/app_grid_configuration.md

```diff
@@ -15,6 +15,7 @@
       "appName": "Twake Chat",
       "icon": "ic_twake_chat.svg",
       "appLink": "https://beta.twake.app/",
+      "publicIconUri": "xxx",
```

Comment:
i think you should add an TO-DO

Reply:
This is config docs so I think no need TO-DO